Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:16.04
 MAINTAINER Michal Rostecki <mrostecki@mirantis.com>
 LABEL Name="virtlet" Version="0.1"
 
-# TODO: dhcpcd5 is only needed for integration tests
+# TODO: dhcpcd5, ebtables and tcpdump are only needed for integration tests
```

Comment:
Looks like `ebtables` are used not only in integration tests, but always in bridge setup.